landscaper/gardenlet/pkg/apis/imports/validation/validation_test.go

```diff
@@ -0,0 +1,336 @@
+				))
+			})
+
+			It("should validate that the replica count is not negative", func() {
+				landscaperGardenletImport.DeploymentConfiguration.ReplicaCount = pointer.Int32Ptr(-1)
+				landscaperGardenletImport.ComponentConfiguration = &gardenletConfiguration
+
+				errorList := ValidateLandscaperImports(landscaperGardenletImport)
+				Expect(errorList).To(ConsistOf(
+					PointTo(MatchFields(IgnoreExtras, Fields{
+						"Type":  Equal(field.ErrorTypeInvalid),
+						"Field": Equal("deploymentConfiguration.replicaCount"),
+					})),
+				))
+			})
+
+			It("should validate that the RevisionHistoryLimit is not negative", func() {
+				landscaperGardenletImport.DeploymentConfiguration.RevisionHistoryLimit = pointer.Int32Ptr(-1)
+				landscaperGardenletImport.ComponentConfiguration = &gardenletConfiguration
+
+				errorList := ValidateLandscaperImports(landscaperGardenletImport)
+				Expect(errorList).To(ConsistOf(
+					PointTo(MatchFields(IgnoreExtras, Fields{
+						"Type":  Equal(field.ErrorTypeInvalid),
+						"Field": Equal("deploymentConfiguration.revisionHistoryLimit"),
+					})),
+				))
+			})
+
+			It("should validate that the service account name is valid", func() {
+				landscaperGardenletImport.DeploymentConfiguration.ServiceAccountName = pointer.StringPtr("x121ä232..")
+				landscaperGardenletImport.ComponentConfiguration = &gardenletConfiguration
+
+				errorList := ValidateLandscaperImports(landscaperGardenletImport)
+				Expect(errorList).To(ConsistOf(
+					PointTo(MatchFields(IgnoreExtras, Fields{
+						"Type":  Equal(field.ErrorTypeInvalid),
+						"Field": Equal("deploymentConfiguration.serviceAccountName"),
+					})),
+				))
+			})
+
+			It("should validate that the pod labels are valid", func() {
+				landscaperGardenletImport.DeploymentConfiguration.PodLabels = map[string]string{"foo!": "bar"}
+				landscaperGardenletImport.ComponentConfiguration = &gardenletConfiguration
+
+				errorList := ValidateLandscaperImports(landscaperGardenletImport)
+				Expect(errorList).To(ConsistOf(
+					PointTo(MatchFields(IgnoreExtras, Fields{
+						"Type":  Equal(field.ErrorTypeInvalid),
+						"Field": Equal("deploymentConfiguration.podLabels"),
+					})),
+				))
+			})
+
+			It("should validate that the podAnnotations are valid", func() {
+				landscaperGardenletImport.DeploymentConfiguration.PodAnnotations = map[string]string{"bar@": "baz"}
+				landscaperGardenletImport.ComponentConfiguration = &gardenletConfiguration
+
+				errorList := ValidateLandscaperImports(landscaperGardenletImport)
+				Expect(errorList).To(ConsistOf(
+					PointTo(MatchFields(IgnoreExtras, Fields{
+						"Type":  Equal(field.ErrorTypeInvalid),
+						"Field": Equal("deploymentConfiguration.podAnnotations"),
+					})),
+				))
+			})
+		})
+
+		Context("validate the backup configuration", func() {
+			It("should validate the backup configuration - do not require landscaperImports.SeedBackupCredentials to be set if Seed is configured with Backup", func() {
```

Comment:
`should validate the backup configuration` is already present in the `Context`, why it is required to copy it in each and every `It`?

```suggestion
			It("should not require landscaperImports.SeedBackupCredentials to be set if Seed is configured with Backup", func() {

```